subprojects/core/src/main/java/org/gradle/profile/ReportGeneratingProfileListener.java

```diff
@@ -35,6 +44,14 @@ public void buildFinished(BuildProfile buildProfile) {
         ProfileReportRenderer renderer = new ProfileReportRenderer();
         File file = new File(buildDir, "reports/profile/profile-" + FILE_DATE_FORMAT.format(new Date(buildProfile.getBuildStarted())) + ".html");
         renderer.writeTo(buildProfile, file);
+        renderReportUrl(file);
+    }
+
+    private void renderReportUrl(File reportFile) {
+        StyledTextOutput textOutput = textOutputFactory.create(ReportGeneratingProfileListener.class, LogLevel.LIFECYCLE);
+        textOutput.println();
```

Comment:
Based on the output I see, we may not need this extra println.

![screen shot 2017-02-17 at 9 21 02 pm](https://cloud.githubusercontent.com/assets/51534/23090253/07253c94-f557-11e6-925f-2868c5c2a5f6.png)